activation/activation.go

```diff
@@ -669,5 +669,5 @@ func (b *Builder) GetPositioningAtxInfo() (types.ATXID, types.LayerID, error) {
 func SignAndFinalizeAtx(signer *signing.EdSigner, atx *types.ActivationTx) error {
 	atx.Signature = signer.Sign(signing.ATX, atx.SignedBytes())
 	atx.SmesherID = signer.NodeID()
-	return atx.Initialize()
+	return nil
```

Comment:
why? are we sure there is no downstream consumer of this atx?

Reply:
reverted